CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## UNRELEASED
 * Feature - Support for provisioning Tasks with ENIs
+* Enhancement - Add support for Docker remote API client version 1.25 [#996](https://github.com/aws/amazon-ecs-agent/pull/996)
```

Comment:
Can you reword this make this message more relevant to customers? Something like:

* Feature - Support running init process in containers by adding support for Docker remote API client version 1.25 [#996](https://github.com/aws/amazon-ecs-agent/pull/996)